conftest.py

```diff
@@ -1,9 +1,17 @@
 import pytest
 import django_webtest
+import os
 
 from django.core.urlresolvers import reverse
+from django.core.files.storage import default_storage
 
 from ideascube.tests.factories import UserFactory
+from ideascube.search.utils import create_index_table
+
+
+@pytest.fixture()
+def cleansearch():
+    create_index_table(True)
```

Comment:
`True`?

Please use the kwarg notation here. Boolean args have this horrible characteristics that they are extremely unexplicit about what they do.